### PR TITLE
enhancing disk setup script

### PR DESCRIPTION
**Pull request by shahnawaz-creator**
## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: docker.local/config/benchmark.yaml [hunk 1]
```diff
@@ -56,7 +66,7 @@ smart_contracts:
     owner_id: 1746b06bb09f55ee01b33b5e2e055d6cc7a900cb57c0a3a5eaabb8a0e7745802
     min_delegates: 0
     max_delegates: 200
-    min_stake: 0.0
+    min_stakke: 0.0
```
Comment (dabasov) [inline docker.local/config/benchmark.yaml]: what is it for?

File: docker.local/config/benchmark.yaml [hunk 2]
```diff
@@ -1,33 +1,43 @@
...
-    - "miner"
+    - "storage"
     - "storage_rest"
+    - "miner"
+    - "miner_rest"
+    - "faucet"
+    - "faucet_rest"
+    - "vesting"
+    - "vesting_rest"
+    - "multi_sig"
```
Comment (dabasov) [inline docker.local/config/benchmark.yaml]: don't need vesting and multi_sig

File: docker.local/config/benchmark.yaml [hunk 3]
```diff
@@ -203,7 +213,7 @@ internal:
   bad: 50ms
   worry: 10ms
   satisfactory: 1ms
-  time_unit: 720h
+  time_unit: 1ms
```
Comment (dabasov) [inline docker.local/config/benchmark.yaml]: why is it changed?

**Review by dabasov — CHANGES_REQUESTED** [review]

File: docker.local/config/benchmark.yaml (hunk 1 above)
Comment (shahnawaz-creator) [reply docker.local/config/benchmark.yaml]: changes reverted

File: docker.local/config/benchmark.yaml (hunk 2 above)
Comment (shahnawaz-creator) [reply docker.local/config/benchmark.yaml]: changes reverted

File: docker.local/config/benchmark.yaml (hunk 3 above)
Comment (shahnawaz-creator) [reply docker.local/config/benchmark.yaml]: changes reverted

**Review by dabasov — APPROVED** [review]
lgtm

**Comment by shahnawaz-creator** [comment]
Cron job added here --> https://github.com/0chain/0helm/pull/255

**Pull request closed without merging**